pages/common/gml2gv.md

```diff
@@ -0,0 +1,17 @@
+# gml2gv
+
+> Convert a graph from the `gml` format to the `gv` format.
+> Converters: `gml2gv`, `gv2gml`, `gv2gxl`, `gxl2gv`, `graphml2gv` & `mm2gv`.
+> More information: <http://graphviz.org/pdf/gml2gv.1.pdf>.
+
+- Convert a graph from the `gml` format to the `gv` format:
+
+`gml2gv -o output.gv input.gml`
```

Comment:
```suggestion
`gml2gv -o {{output.gv}} {{input.gml}}`
```

Reply:
Thank you. Missed that. Harder to spot in the template.